service/src/ocall_bridge/ffi/send_to_parentchain.rs

```diff
@@ -25,38 +25,25 @@ use std::{slice, sync::Arc, vec::Vec};
 ///
 /// FFI are always unsafe
 #[no_mangle]
-pub unsafe extern "C" fn ocall_send_block_and_confirmation(
+pub unsafe extern "C" fn ocall_send_to_parentchain(
 	confirmations: *const u8,
 	confirmations_size: u32,
-	signed_blocks_ptr: *const u8,
-	signed_blocks_size: u32,
 ) -> sgx_status_t {
-	send_block_and_confirmation(
-		confirmations,
-		confirmations_size,
-		signed_blocks_ptr,
-		signed_blocks_size,
-		Bridge::get_oc_api(),
-	)
+	send_to_parentchain(confirmations, confirmations_size, Bridge::get_oc_api())
 }
 
-fn send_block_and_confirmation(
+fn send_to_parentchain(
 	confirmations: *const u8,
```

Comment:
since these have been renamed in `woker_on_chain_ocall.rs`, should we also rename them to `extrinsics_encoded ` here? Or it that taking it too far?